Makefile

```diff
@@ -83,23 +84,21 @@ fmt:
 
 .PHONY: lint
 lint:
-	$(GOVET) $(TOP_PKGS)
+	$(GOVET) $(ALL_PKGS)
 	@cat /dev/null > $(LINT_LOG)
-	@$(foreach pkg, $(TOP_PKGS), $(GOLINT) $(pkg) | grep -v -e pkg/es/wrapper.go -e /mocks/ -e thrift-gen -e thrift-0.9.2 >> $(LINT_LOG) || true;)
+	@$(foreach pkg, $(ALL_PKGS), $(GOLINT) $(pkg) | grep -v -e pkg/es/wrapper.go -e /mocks/ -e thrift-gen -e thrift-0.9.2 >> $(LINT_LOG) || true;)
 	@[ ! -s "$(LINT_LOG)" ] || (echo "Lint Failures" | cat - $(LINT_LOG) && false)
 	@$(GOFMT) -e -s -l $(ALL_SRC) > $(FMT_LOG)
 	@./scripts/updateLicenses.sh >> $(FMT_LOG)
 	@[ ! -s "$(FMT_LOG)" ] || (echo "Go fmt or license check failures, run 'make fmt'" | cat - $(FMT_LOG) && false)
 
-.PHONY: install-glide
-install-glide:
-	# all we want is: glide --version || go get github.com/Masterminds/glide
-	# but have to pin to 0.12.3 due to https://github.com/Masterminds/glide/issues/745
-	@which glide > /dev/null || (mkdir -p $(GOPATH)/src/github.com/Masterminds && cd $(GOPATH)/src/github.com/Masterminds && git clone https://github.com/Masterminds/glide.git && cd glide && git checkout v0.12.3 && go install)
+.PHONY: install-dep
+install-dep:
+	@which dep > /dev/null || (curl -L -s -o dep https://github.com/golang/dep/releases/download/$(DEP_VER)/dep-linux-amd64 && chmod +x dep && sudo mv dep /usr/local/bin/)
```

Comment:
Should this download link take into account host platform?

Reply:
yea good point